src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const listArrays = sourceString
+    .split(';')
+    .filter((item) => item.length)
+    .map((element) => element.split(':').map((item) => item.trim()));
+
+  const resultObject = {};
+  const filterArray = listArrays.reduce((acc, value) => {
+    const [key, val] = value;
+
+    acc[key] = val;
+
+    return acc;
```

Comment:
The variable `filterArray` does not actually contain an array, so the name is misleading. Since it's the final object with the CSS styles, a more appropriate name would be something like `styleObject` or `cssStyles`.

Reply:
ok